applications/nrf_desktop/README.rst

```diff
@@ -74,47 +85,22 @@ The project supports the following types:
 
 Build type is specified using the cmake variable CMAKE_BUILD_TYPE.
 
-
-Application configuration
-*************************
-
-For detailed information about the application configuration and how to port
-it to a different hardware platform, see the following pages.
-
-.. toctree::
-    :maxdepth: 1
-
-    doc/porting_guide.rst
-
-
-Firmware architecture
-*********************
-
-The nRF Desktop application design aims at configurability and extensibility
-while being responsive.
-
-The application architecture is modular and event-driven.
-This means that parts of application functionality were separated into isolated modules
-that communicate with each other using application events.
-Application events are handled by the :ref:`event_manager`. Modules register
-themselves as listeners of events they need to react on.
-
-The application limits the number of threads used to the minimum and does not
-use user-space threads. Most of the application activity happens in the
-context of the system workqueue thread via scheduled work objects or the event
-manager callbacks (executed from the system workqueue thread).
-
-Firmware modules:
-
-.. toctree::
-    :maxdepth: 1
-
-    doc/hw_interface.rst
-    doc/modules.rst
-
-
 Testing
-*******
+-------
 
 The code is currently work-in-progress and as such is not fully functional,
```

Comment:
This section needs a different type of content but since this is a PR for the structure, we can leave this.